pyconfyg/pyconfyg.py

```diff
@@ -0,0 +1,113 @@
+import ast
+import copy
+import functools
+import itertools
+import re
+import traceback
+
+import astunparse
+
+
+def update_ast(tree, overwrite, allow_double_assignation=False, allow_tuple_assignation=False):
+    met_targets = []
+    for node in tree.body:
+        if not isinstance(node, ast.Assign):
+            continue
+        for target in node.targets:
+            if hasattr(target, "id"):
+                assert allow_tuple_assignation or isinstance(target, ast.Name), \
```

Comment:
Would rather not perform `assertions` inside your code, but would check some conditions and raise a custom exception where needed.